scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -672,6 +672,10 @@
                     'different test runs. This flag is designed for human '
                     'consumption, and does not have any impact within the '
                     'system.')
+flags.DEFINE_boolean('auto_mixed_precision', False,
+                     'enable/disable grappler optimization that implements '
+                     'automatic mixed precision support. Enabling automatic '
```

Comment:
Add a sentence stating: "This is equivalent to --use_fp16, but uses a different implementation".